packages/cozy-konnector-libs/src/libs/categorization/localModel/classifier.js

```diff
@@ -166,7 +170,8 @@ const createClassifier = async options => {
 
   log('debug', 'Instanciating a new classifier')
 
-  const classifierOptions = getClassifierOptions(transactions)
+  const nbUniqueCategories = getUniqueCategories(transactions)
+  const classifierOptions = getClassifierOptions(nbUniqueCategories.length)
```

Comment:
You might want to have more defensive code, e.g. `getClassifierOptions(a?.length || 0)`